TAF/utils/scripts/snap/deploy-edgex.sh

```diff
@@ -1,2 +1,39 @@
-#!/bin/sh
+#!/bin/bash
+
+TEST_STRATEGY=${1:-}
+APPSERVICE=${2:-}
+CONF_DIR=/custom-config
+
+sudo snap remove edgexfoundry
+sudo snap remove edgex-device-modbus
+sudo snap install edgexfoundry --channel=2.0/stable
+sudo snap install edgex-device-modbus --channel=2.0/stable
```

Comment:
For your final PR, it would be nice if the TAF snap support allowed you to specify a Snap Store channel or a local path for the EdgeX snaps.

Also shouldn't you be installing app-service-configurable too?

Reply:
Fixed.